### PR TITLE
[#44954] Submit button display text

### PR DESCRIPTION
**Pull request by Kharonus**
- https://community.openproject.org/work_packages/44954
- added switch for three different states:
  - no selection
  - singular selection
  - plural selection

### CODE REVIEW

File: frontend/src/app/shared/components/storages/file-picker-modal/file-picker-modal.component.ts [hunk 1]
```diff
@@ -71,7 +71,16 @@ export class FilePickerModalComponent extends OpModalComponent implements OnInit
...
-      submit: ():string => this.i18n.t('js.storages.file_links.selection_any', { number: this.selectedFileCount }),
+      submit: (count:number):string => {
+        switch (count) {
+          case 0:
+            return this.i18n.t('js.storages.file_links.selection_none');
+          case 1:
+            return this.i18n.t('js.storages.file_links.selection_one');
+          default:
+            return this.i18n.t('js.storages.file_links.selection_any', { number: count });
+        }
```
Comment (b12f) [inline frontend/src/app/shared/components/storages/file-picker-modal/file-picker-modal.component.ts]: Does this pattern work for all languages? The i18n module itself might have some more indications, I think some language require and extra case for e.g. 2

Comment (Kharonus) [reply frontend/src/app/shared/components/storages/file-picker-modal/file-picker-modal.component.ts]: yes, you are right. russian for example has own plural forms for count 2 to 4. Not sure though, how we should solve this now. I checked other parts of the application, and in so many places we have only two options: singular and plural.

any hints?

Comment (Kharonus) [reply frontend/src/app/shared/components/storages/file-picker-modal/file-picker-modal.component.ts]: fixed after talking to @oliverguenther

**Review by b12f — APPROVED** [review]
LGTM

**Pull request merged**